include/HubFramework/HUBViewController.h

```diff
@@ -133,8 +133,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, nullable, readonly) id<HUBViewModel> viewModel;
 
-/// The views of the root body components that are currently visible on screen, keyed by their index paths
-@property (nonatomic, strong, readonly) NSDictionary<NSIndexPath *, UIView *> *visibleBodyComponentViewIndexPaths;
+/**
+ *  The views of the body components that are currently visible on screen, keyed by their index path
+ *
+ *  The index paths used for keys contains the indexes for the components' views, starting from the root. For example,
+ *  a root body component at index 4 will have an index path with just the index 4, while the 3rd child of that component
```

Comment:
Wording is kind of confusing as the first example says "index 4", whereas the second example "3rd child". Perhaps replace 3rd child with "child at index 2"? :smile: